tshock/bootstrap.sh

```diff
@@ -17,20 +17,25 @@ if [ $STORAGETYPE = "mysql" ]; then
 fi
 
 WORLD_PATH="/root/.local/share/Terraria/Worlds/$WORLD_FILENAME"
+if [ ! -f "/config/serverconfig.txt" ]; then
+    echo "Server configuration not found, running with default server configuration."
+    echo "Please ensure your desired serverconfig.txt file is volumed into docker: -v <path_to_config_file>:/config"
+    cp ./serverconfig-default.txt /config/serverconfig.txt
+fi
 
 if [ -z "$WORLD_FILENAME" ]; then
   echo "No world file specified in environment WORLD_FILENAME."
-  if [ -z "$@" ]; then 
+  if [ -z "$@" ]; then
     echo "Running server setup..."
   else
     echo "Running server with command flags: $@"
   fi
-  mono --server --gc=sgen -O=all TerrariaServer.exe -configpath "$CONFIGPATH" -logpath "$LOGPATH" "$@" 
+  mono --server --gc=sgen -O=all TerrariaServer.exe -configpath "$CONFIGPATH" -logpath "$LOGPATH" -config "$CONFIG" "$@"
```

Comment:
If the `$CONFIG` variable is not set, will the server still launch?  Maybe we should default the value for the `$CONFIG` variable?

Reply:
It is defaulted to `ENV CONFIG=/config/serverconfig.txt` line 32 tshock/Dockerfile currently, will update it to default to the `Worlds` dir of course.